src/os/bluestore/BlueFS.cc

```diff
@@ -606,6 +615,181 @@ void BlueFS::_stop_alloc()
   }
 }
 
+bool is_all_zeros(const char* p, size_t s)
```

Comment:
Just realized that there is mem_is_zero() in inline_memory.h

And just to let you know there is buffer::ptr::is_zero() method which might be of some help too....

Reply:
I think I will switch to `mem_is_zero()`, thanks!
I would need to chop buffers to use `buffer::ptr::is_zero()`, so I will skip on that one.